Sources/WhisperKit/Core/Models.swift

```diff
@@ -262,6 +263,7 @@ public struct DecodingOptions {
                 supressTokens: [Int]? = nil,
                 compressionRatioThreshold: Float? = 2.4,
                 logProbThreshold: Float? = -1.0,
+                firstTokenLogProbThreshold: Float? = nil,
```

Comment:
what could be possibly a good default value here @ZachNagengast ?

Reply:
Low-effort: Default to the value right above this line: `logProbThreshold: Float? = -1.0`.
Mid-effort: Calibrate based on the few hallucination samples from https://github.com/openai/whisper/discussions/1873 as @ZachNagengast  shared earlier on Discord.